### DISCUSSION TITLE
Why not make a skin just a directory instead of a compiled DLL?

### DISCUSSION THREAD

**Discussion by HubKing** [Open-Shell/Open-Shell-Menu#734] [Q&A]
So, I want to change the font size of the default skin (enabling "Large Font" did not work). I read the page, and it says that I have to edit it with Resource Hacker. This is a really cumbersome way to edit the settings text. I need to create a copy of it, rename the extension, open it with RH, compile it, rename the extension again, move it to the skin directory.

Why not just make each resource a directory that contains a plain text file for the skin settings? Or at least have an option to load an "unpacked skin" just as web browsers allow loading unzipped extension? Is there any reason to put all the files into a compiled DLL file?

**Comment by bonzibudd** [comment]
If I had to guess, it was probably done that way in order to make it easier to distinguish individual skins and have them be more portable for people who make 3rd party skins. I also wish there was an easier way to compile the skins, and I don't think it would be too difficult because it's a simple format. Regarding having a directory system, I don't really think that would be too much better, because it would still have the same permissions issues and would require a lot of copying back and forth between directories.
